webapp/security/views.py

```diff
@@ -268,8 +268,53 @@ def api_create_notice():
 # ===
 def cve_index():
 
-    list_cve = db_session.query(CVE).limit(10)
-    return flask.render_template("security/cve/index.html", list_cve=list_cve)
+    page = flask.request.args.get("page", default=1, type=int)
+    order_by = flask.request.args.get("order", type=str)
+    search_id = flask.request.args.get("id", type=str)
+    search_desc = flask.request.args.get("search_desc", type=str)
+
+    cves_query = db_session.query(CVE)
+
+    page_size = 3
+    total_results = cves_query.count()
+    total_pages = ceil(total_results / page_size)
+    offset = page * page_size - page_size
+
+    if page < 1 or 1 < page > total_pages:
+        flask.abort(404)
+
+    sort = asc if order_by == "oldest" else desc
+
+    # Search functionality
+
+    if search_id:
```

Comment:
what is `search_id`?

Reply:
@carkod Let's have a quick catch-up after the standup this morning.